### PR TITLE
Ember upgrade 2.7

### PR DESCRIPTION
**Pull request by hbrysiewicz**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 DockYard, Inc
+Copyright (c) 2016
```
Comment (martndemus) [inline LICENSE.md]: Restore `DockYard, Inc` please :)

File: addon/helpers/one-way-select/contains.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
 import Ember from 'ember';
 
-const { isArray, isPresent, get } = Ember;
+const {
+  A,
```
Comment (martndemus) [inline addon/helpers/one-way-select/contains.js]: Alias as `emberArray` please!

File: config/ember-try.js [hunk 3]
```diff
@@ -17,72 +19,6 @@ module.exports = {
       }
     },
     {
```
Comment (martndemus) [inline config/ember-try.js]: Restore this, or use the new style config: https://github.com/ember-cli/ember-try#config

File: package.json [hunk 4]
```diff
@@ -23,31 +23,32 @@
...
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-qunit": "^2.1.0",
+    "ember-cli-release": "0.2.9",
     "ember-cli-sri": "^2.1.0",
+    "ember-cli-test-loader": "^1.1.0",
+    "ember-data": "^2.7.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
-    "ember-hash-helper-polyfill": "^0.1.1",
```
Comment (martndemus) [inline package.json]: Restore this one.

File: package.json [hunk 5]
```diff
@@ -23,31 +23,32 @@
...
-    "ember-cli-qunit": "^1.4.0",
-    "ember-cli-release": "0.2.8",
+    "ember-cli-jshint": "^1.0.0",
+    "ember-cli-qunit": "^2.0.0",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-qunit": "^2.1.0",
+    "ember-cli-release": "0.2.9",
     "ember-cli-sri": "^2.1.0",
+    "ember-cli-test-loader": "^1.1.0",
+    "ember-data": "^2.7.0",
```
Comment (martndemus) [inline package.json]: remove `ember-data`

File: package.json [hunk 6]
```diff
@@ -23,31 +23,32 @@
...
+    "ember-suave": "4.0.0",
     "loader.js": "^4.0.1"
   },
   "keywords": [
     "ember-addon"
   ],
   "dependencies": {
     "ember-cli-babel": "^5.1.6",
-    "ember-cli-htmlbars": "^1.0.3",
-    "ember-get-helper": "1.0.4",
```
Comment (martndemus) [inline package.json]: Restore this one

**Comment by martndemus** [comment]
You also might need to do the following:
https://github.com/martndemus/ember-form-for/commit/189f5d234bb6886874c6e463965f4088732752c1
Because the `beta` and `canary` channels are broken.

File: addon/helpers/one-way-select/contains.js (hunk 2 above)
Comment (hbrysiewicz) [inline addon/helpers/one-way-select/contains.js]: I disagree - maybe this should be discussed in bike shed Friday talks?

Comment (rwjblue) [inline addon/helpers/one-way-select/contains.js]: The issue with `A` is that JSHint complains about invoking a function starting with a capital letter without `new` keyword.

Comment (martndemus) [inline addon/helpers/one-way-select/contains.js]: regardless, `emberArray` is much clearer of it's intentions than `A`

Comment (hbrysiewicz) [inline addon/helpers/one-way-select/contains.js]: Thanks @rwjblue that explains it much better

File: package.json (hunk 4 above)
Comment (hbrysiewicz) [inline package.json]: Ember says it is no longer necessary

screen shot 2016-07-28 at 11 24 38 am

File: package.json (hunk 6 above)
Comment (hbrysiewicz) [inline package.json]: `ember-get-helpers` is also no longer needed

Comment (martndemus) [inline package.json]: It's needed for `1.13`

File: package.json (hunk 4 above)
Comment (martndemus) [inline package.json]: It's necessary for ember `1.13`

**Comment by martndemus** [comment]
LGTM now!

**Comment by martndemus** [comment]
Looks like the beta and canary channels are failing as predicted. Can you remove these two lines as temp fix? https://github.com/DockYard/ember-one-way-controls/blob/master/.travis.yml#L30-L31

**Comment by hbrysiewicz** [comment]
I think there is actually a way in the config to allow for those two to fail and still pass travis... let me investigate @martndemus

**Comment by martndemus** [comment]
Oh, right, you can move beta to allowed failures in `.travis.yml`.

**Pull request merged**